README.md

```diff
@@ -1,8 +1,33 @@
-# NEAR BOS Web Component ( custom element )
+<!-- markdownlint-disable MD014 -->
 
-This is a Proof of Concept of embedding a NEAR BOS widget into any web application as a Web Component / Custom element.
+<div align="center">
 
-Just load react production react bundles into your index.html as shown below, and use the `near-social-viewer` custom element to embed the BOS widget.
+  <h1>near custom element</h1>
+
+  <p>
+    <strong>Easily embed a <a href="https://near.social" target="_blank">near social widget</a> into any web app and deploy to <a href="https://web4.near.page/" target="_blank">web4</a>.</strong>
+  </p>
+
+</div>
+
+`near-social-viewer` is a [web component (custom element)](https://developer.mozilla.org/en-US/docs/Web/API/Web_components) that implements the [near-social-vm](https://github.com/NearSocial/VM) for rendering code stored on-chain in the [SocialDB](https://github.com/NearSocial/social-db) smart contract (social.near). It is the simplest way to create your own [near social viewer](https://github.com/NearSocial/viewer) and it is the easiest method for embedding [Widgets](https://thewiki.near.page/near.social_widgets) into any web application.
+
+<strong>The latest version of this package is deployed to [nearfs](https://github.com/vgrichina/nearfs) at:</strong>
```

Comment:
should we also mention that it is published to NPM? And also having this nearfs link means we have to make sure to update it for every release. How do we automate that in a good way? Should we rather publish releases with a file pointing to the NEARFS link?

Reply:
This is a good idea. 

Do we need the full web4 in the address or can we just do the cid?

And is it right to think this should go in package.json, like this?

```
"nearfs": { // or ipfs?
    "cid": "..."
}
```

Reply:
@petersalomonsen pull request for this here: #39